bazel/README.md

```diff
@@ -112,12 +112,29 @@ for how to update or override dependencies.
     consider uninstalling binutils.
 
     ### Windows
-    On Windows, you'll need to install several dependencies manually.
 
-    [python3](https://www.python.org/downloads/): Specifically, the Windows-native flavor. The POSIX flavor
-    available via MSYS2 will not work, nor will the Windows Store flavor. You need to add a symlink for `python3.exe` pointing to
-    the installed `python.exe` for Bazel rules which follow POSIX conventions. Be sure to add
-    `pip.exe` to the PATH and install the `wheel` package.
+    Follow all items below to configure the Windows build toolchain and prepare the build environment,
+    or just install Docker, Git, MSYS2 to use the Docker image (the same as used in the CI pipeline):
+    ```
+    ./ci/run_envoy_docker_windows.sh './ci/windows_ci_steps.sh'
+    ```
```

Comment:
It looks like this method already has its own section "Building Envoy with the CI Docker image", but is lacking the extra dependency information. Is this supposed to be a recommendation that developers use the docker image?

Reply:
I was thinking we add a note at the top of the "bazel quick start" section that links to the docker section so people who want to use docker can short circuit all the toolchain setup (and also add the dependencies needed for docker down there, will push a change that reflects that